src/gt4py/cli.py

```diff
@@ -75,32 +103,41 @@ def _convert_value(
         else:
             return type_spec(value)
 
+    def _try_split(self, value: str) -> Tuple[str, str]:
```

Comment:
There is no benefit to it being a static method (it shouldn't be called without an instance since it is `protected` and also it is meaningless outside the context of this class).